types/artifacts.d.ts

```diff
@@ -675,6 +677,13 @@ declare global {
         observedSpeedIndex: number;
         observedSpeedIndexTs: number;
       }
+
+      export interface Form {
```

Comment:
One of the audits to follow from this will be a check that input elements have an associated label. So, it follows that the artifact needs to be able to represent that relationship.

On the page, `inputEl.labels` returns a list of label elements associated with the input element. Much like `.form`, we can use this.

Instead of `labels` being a property next to `inputs`, it should probably be within `inputs`.

Reply:
(decided against this offline. we have an axe audit that tests for this already so not needed)